src/components/hover-card.vue

```diff
@@ -53,50 +50,6 @@ const props = defineProps({
   }
 });
 
-const width = ref('');
```

Comment:
got rid of everything where hovercard is responsible for computing its own size and position

Reply:
I reverted this code so the resizing behavior is back. It seems to work (truncation behavior in Swahili looks good). There is some issue with the tests, though, and checking the width. Could it have to do with not having the `popover` wrapper? The popover wrapper didn't work in this test anymore because of the v-if visibility check.